src/index.ts

```diff
@@ -39,6 +39,7 @@ export interface CoreOptions {
   multipart?: RequestPart[];
   forever?: boolean;
   pool?: HttpsAgentOptions | HttpAgentOptions;
+  followRedirect?: boolean;
```

Comment:
for full `request` compatibility, `followRedirect` also supports a function:
```
followRedirect?: boolean | ((res: f.Response) => boolean);
```
see https://github.com/request/request/blob/536f0e76b249e4545c3ba2ac75e643146ebf3824/lib/redirect.js#L25